tools/mo/openvino/tools/mo/main.py

```diff
@@ -350,7 +348,8 @@ def emit_ir(graph: Graph, argv: argparse.Namespace):
                     mean_data=mean_data,
                     input_names=input_names,
                     meta_info=get_meta_info(argv),
-                    use_temporary_path=True)
+                    use_temporary_path=True,
+                    convert_types=argv.legacy_ir_generation)
```

Comment:
please do not change behavior of deprecated option `legacy_ir_generation`. Please revert